experiment/build/gcb_build.py

```diff
@@ -59,6 +68,25 @@ def _build_benchmark_coverage(benchmark: str) -> Tuple[int, str]:
     return _build(config_file, config_name, substitutions)
 
 
+def _add_builder_arguments_to_config(base: str, fuzzer: str) -> str:
+    """If there are fuzzer-specific arguments, make a config file with them."""
+    fuzzer_config = fuzzer_config_utils.get_by_variant_name(fuzzer)
+    if 'builder_arguments' not in fuzzer_config:
+        return base
+
+    # TODO(mbarbella): Rather than rewrite yaml files, use the GCB API.
+    args = fuzzer_config['builder_arguments']
+    config = yaml_utils.read(base)
+    for step in config['steps']:
+        if 'id' in step and step['id'] in BUILDER_STEP_IDS:
+            # Append additional flags before the final argument.
+            step['args'] = step['args'][:-1] + args + [step['args'][-1]]
+
+    new_config_path = tempfile.NamedTemporaryFile().name
```

Comment:
Where do we clean this temp file.

Reply:
Hmm. We don't, and there isn't a particularly clean way to handle cleaning it up with this implementation (since sometimes the yaml file is the default one that we didn't modify). Though writing to the configs directory wasn't too clean either, I don't think there are actually any conflicts so I'll return to that.

Reply:
Done